doc/integrator/create_application.rst.tmpl

```diff
@@ -61,7 +61,7 @@ To create the application, first apply the ``c2cgeoportal_create`` scaffold:
     docker run --rm -ti --volume=$(pwd):/src \
         camptocamp/geomapfish-tools:${GEOMAPFISH_VERSION} \
         run $(id -u) $(id -g) /src \
-        pcreate -s c2cgeoportal_create ${GEOMAPFISH_PROJECT}
+        pcreate -s c2cgeoportal_create ${DOLLAR}{GEOMAPFISH_PROJECT}
```

Comment:
I do not understand the DOLLAR, since the user will probably not have set it, this will do nothing. And why should the user set DOLLAR environment variable anyway?
Maybe I misunderstand.

Reply:
`${GEOMAPFISH_PROJECT}` => ``
`${DOLLAR}{GEOMAPFISH_PROJECT}` => `${GEOMAPFISH_PROJECT}`
for the end user, you can verify in the current documentation :-)